tests/Doctrine/Tests/Common/Annotations/DocParserTest.php

```diff
@@ -486,6 +496,28 @@ public function getAnnotationVarTypeProviderInvalid()
             array('annotation', AnnotationTargetAll::class, '{"str"}','array'),
             array('annotation', AnnotationTargetAll::class, '{1,2,3,4}','array'),
             array('annotation', AnnotationTargetAll::class, '@Name','an instance of Doctrine\Tests\Common\Annotations\Name'),
+
+            // unqualified annotation instance
+            array('unqualifiedAnnotation', 'AnnotationTargetAll', '@Name','an instance of Doctrine\Tests\Common\Annotations\Name'),
+
+            // renamed annotation instance
+            array('renamedAnnotation', 'RenamedAnnotationTargetAll', '@Name','an instance of Doctrine\Tests\Common\Annotations\Name'),
+
+            // partially namespaced annotation instance
+            array(
+                'partiallyNamedAnnotation',
+                'Common\Annotations\Fixtures\AnnotationTargetAll',
```

Comment:
This assumption is strange, it should be the FQCN, not the one used in declaration. It works fine with arrays - see arrayOf* tests below, but for non-array values, it doesn't seem to be correct...